guides/source/configuring.md

```diff
@@ -1507,6 +1507,10 @@ The default value depends on the `config.load_defaults` target version:
 | (original)            | `true`               |
 | 7.0                   | `false`              |
 
+#### `config.action_view.prepend_content_exfiltration_prevention`
```

Comment:
Could you add a table outlining the default versions, similar to other settings?